Libraries/Components/MapView/MapView.js

```diff
@@ -252,8 +260,12 @@ var MapView = React.createClass({
     }
   },
 
+  _onAnnotationDragChange: function(event: Event) {
+    this.props.onAnnotationDragChange && this.props.onAnnotationDragChange(event);
+  },
+
   render: function() {
-    return <RCTMap {...this.props} onPress={this._onPress} onChange={this._onChange} />;
+    return <RCTMap {...this.props} onPress={this._onPress} onChange={this._onChange} onAnnotationDragChange={this._onAnnotationDragChange} />;
```

Comment:
This is long enough now that we should wrap the props:

    return (
      <RCTMap
        {...this.props}
        onPress={this._onPress}
        onChange={this._onChange}
        onAnnotationDragChange={this._onAnnotationDragChange}
      />
    );